### DISCUSSION TITLE
How to change the arrow style of a LEADER?

### DISCUSSION THREAD

**Discussion by luoyu-123** [mozman/ezdxf#321] [Q&A]
Hello, respected Mr. Mozman, I want to change the arrow style when using the add_leader() interface function, but no matter how I pass in the override parameter ('dimblk','dimblk1','dimblk2') these three None of the results obtained by the attributes took effect.
It turns out that the arrow style cannot be changed：
![image](https://user-images.githubusercontent.com/59160360/105275559-98d49980-5bda-11eb-8bf4-ae2d08c32566.png)，
I want to get the style of the arrow：
![image](https://user-images.githubusercontent.com/59160360/105275495-7773ad80-5bda-11eb-9197-bf06a6f750d7.png)

**Comment by mozman** [comment]
You have to override the dimvar `dimldrblk`.
